PokerGame/PokerEngine/Poker.cs

```diff
@@ -115,29 +117,52 @@ private static List<Card> CalculateHandWithHighestCardAndKicker(List<Card> handO
             return null;
         }
 
-        private static bool HasPair(List<Card> hand, out Rank pair)
+        private static Rank CalculateHighestPair(Rank pairOneRank, Rank pairTwoRank)
         {
-            for (var i = 0; i < hand.Count; i++)
+            var hand = new List<Card>()
             {
-                var subHand = hand.GetRange(i + 1, hand.Count - i - 1);
-                foreach (Card card in subHand)
+                new Card() { Rank = pairOneRank },
+                new Card() { Rank = pairTwoRank }
+            };
+            var highestPairRank = HighestCard(hand).Rank;
```

Comment:
Surely much simpler to just compare the two ranks?
```
if (pairOneRank > pairTwoRank)
{
    return pairOneRank;
} 
else if (pairTwoRank > pairOneRank)
{
    return pairTwoRank;
}
else
{
    return null;
}
```

Reply:
yes :)